Mathlib/Tactic/Core.lean

```diff
@@ -90,8 +90,8 @@ def getUsingArg : Syntax → TacticM Syntax
   | _                     => Elab.throwUnsupportedSyntax
 
 /--
-`repeat1 tac` applies `tac` to main goal at least once. If the application succeeds,
-the tactic is applied recursively to the generated subgoals until it eventually fails.
+`repeat1 tac` repeatedly applies `tac` until it fails.
+`repeat1` succeeds if and only if `tac` succeeds at least once.
```

Comment:
@digama0 this is the doc-string you corrected in a previous comment. I'd only moved it from here, but now that you've provided the `repeat'` combinator in `Std` there's no need to make a new file or move anything around, so instead I'm just correcting the old doc-string in place.